packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -370,8 +370,7 @@ export class ConnectionManager implements IConnectionManager {
         this._forceReadonly = readonly;
 
         if (oldValue !== this.readonly) {
-            assert(this._reconnectMode !== ReconnectMode.Never,
-                0x279 /* "API is not supported for non-connecting or closed container" */);
+            throw new UsageError("API is not supported for non-connecting or closed container");
```

Comment:
This was conditional before, now it became non-conditional error. You skipped this._reconnectMode !== ReconnectMode.Never condition (and that's likely why your tests failed). It should be 

```ts
if (this._reconnectMode === ReconnectMode.Never) {
    throw new UsageError("API is not supported for non-connecting or closed container");
}
Consider running tests locally to find these issues on your machine - it will allow you to faster iterate and resolve any issues.